types/react-native-incall-manager/index.d.ts

```diff
@@ -0,0 +1,64 @@
+// Type definitions for react-native-incall-manager 3.2
+// Project: https://github.com/zxcpoiu/react-native-incall-manager#readme
+// Definitions by: Carlos Quiroga <https://github.com/KarlosQ>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
+
+export interface StartSetup {
+    media?: string;
+    auto?: boolean;
+    ringback?: string;
+}
+
+export interface StopSetup {
+    busytone?: string;
+}
+
+export default class InCallManager {
```

Comment:
[Looks like the default is an _instance_ of the class, not the class itself.](https://github.com/zxcpoiu/react-native-incall-manager/blob/master/index.js#L187) - i'd remove the `export default` from this class, remove the `static` modifier from all the methods, then add
```ts
declare const _default: InCallManager;
export default _default;
```

Reply:
Thanks you for the info 👍